src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -99,4 +116,57 @@ private void copyUrl() {
         url.putString(USERGUIDE_URL);
         clipboard.setContent(url);
     }
+
+    /**
+     * Initializes the command summary table and wraps the text in the cells.
+     */
+    private void initializeCommandSummary() {
+        commandSummaryList.addAll(
+                new CommandSummary("Add", "add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/AGE s/SEX "
+                        + "ap/APPOINTMENT t/TAG\ne.g., add n/Evie Sage p/88888888 e/eviesage@example.com "
+                        + "a/Hickory Forest b/23 s/Female ap/11/11/2024 1100"),
+                new CommandSummary("Clear", "clear"),
+                new CommandSummary("Delete", "delete INDEX / delete NAME\ne.g., delete 3, delete Alex Yeoh"),
+                new CommandSummary("Edit", "edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/AGE] [s/SEX] "
+                        + "[ap/APPOINTMENT] [t/TAG]\ne.g., edit John Doe n/Betsy Crower t/ ap/"),
+                new CommandSummary("Find", "find KEYWORD [MORE_KEYWORDS]\ne.g., find olive 87438"),
+                new CommandSummary("Help", "help"),
+                new CommandSummary("List", "list")
+        );
+
```

Comment:
Good to have [square brackets] to indicate optional field! Do we need a side note on that? an exaplanation of what square bracket, UPPERCASE means?

Reply:
I just copied from the changes made in User Guide. I will update the command summary when the User Guide on website made changes. I think we should discuss the format. Then, I will add side note accordingly.